proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+# dotnet nuget trust command
+
```

Comment:
Overall comment on help: 

System.CommandLine may be displaying your help (ask Sarah), in which case some of this will just be fixed. 

There are some inconsistencies in Help here with the rest of the CLI. For example, the section headers are not all caps in the CLI. Try a command like `dotnet build` (we have an issue on the wide first column)

Reply:
Yeah, that part is fine. The help format here is mostly for informative purposes. Things don't need to be pixel-perfect.